### PR TITLE
Update to 3.7.3

### PR DESCRIPTION
**Pull request by deepsworld**
Checklist
* [X ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ X] Bumped the build number (if the version is unchanged)
* [ X] Reset the build number to `0` (if the version changed)
* [X ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by deepsworld** [comment]
@conda-forge-admin, please rerender

**Comment by deepsworld** [comment]
I guess gnutls needs to be updated too as it is under export run test.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ requirements:
     - autoconf
     - m4
     - make
+    - binutils >=2.36
```
Comment (erykoff) [inline recipe/meta.yaml:28]: I can't comment on lines above, but maybe `    - {{ pin_subpackage('nettle', max_pin='x.x') }}` needs to be `    - {{ pin_subpackage('nettle', max_pin='x.x.x') }}`

**Comment by sachalau** [comment]
Closing since more recent version pushed

**Pull request closed without merging**